### PR TITLE
Gs new refactor format

### PR DESCRIPTION
**Pull request by JayFoxRox**
Temporary to solve clang-format issues

### CODE REVIEW

File: src/video_core/shader/shader_jit_x64.h [hunk 1]
```diff
@@ -84,19 +84,32 @@ class JitShader : public Gen::XCodeBlock {
...
+     * `CALL` instruction.
      */
     void Compile_Return();
 
     BitSet32 PersistentCallerSavedRegs();
 
     /**
-     * Assertion evaluated at compile-time, but only triggered if executed at runtime.
+     * Assertion evaluated at compile-time, but only triggered if executed at
+     * runtime.
```
Comment (JayFoxRox) [inline src/video_core/shader/shader_jit_x64.h]: bad

File: src/video_core/shader/shader_jit_x64.h [hunk 2]
```diff
@@ -84,19 +84,32 @@ class JitShader : public Gen::XCodeBlock {
     void Compile_UniformCondition(Instruction instr);
 
     /**
-     * Emits the code to conditionally return from a subroutine envoked by the `CALL` instruction.
+     * Emits the code to conditionally return from a subroutine envoked by the
+     * `CALL` instruction.
```
Comment (JayFoxRox) [inline src/video_core/shader/shader_jit_x64.h]: bad

File: src/video_core/command_processor.cpp [hunk 3]
```diff
@@ -300,13 +300,78 @@ static void WritePicaReg(u32 id, u32 value, u32 mask) {
             };
 
             primitive_assembler.SubmitVertex(output_vertex, AddTriangle);
-        }
+    }
```
Comment (JayFoxRox) [inline src/video_core/command_processor.cpp]: bad

File: src/video_core/command_processor.cpp [hunk 4]
```diff
@@ -300,13 +300,78 @@ static void WritePicaReg(u32 id, u32 value, u32 mask) {
...
-            g_debug_context->recorder->MemoryAccessed(Memory::GetPhysicalPointer(range.first),
-                                                      range.second, range.first);
-        }
+    for (auto& range : memory_accesses.ranges) {
+        g_debug_context->recorder->MemoryAccessed(Memory::GetPhysicalPointer(range.first),
+                                                  range.second, range.first);
+    }
+
+    break;
+    }
```
Comment (JayFoxRox) [inline src/video_core/command_processor.cpp]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 5]
```diff
@@ -47,18 +48,23 @@ class DebugContext {
     };
 
     /**
-     * Inherit from this class to be notified of events registered to some debug context.
+     * Inherit from this class to be notified of events registered to some debug
+     * context.
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 6]
```diff
@@ -47,18 +48,23 @@ class DebugContext {
...
      * Most importantly this is used for our debugger GUI.
      *
-     * To implement event handling, override the OnPicaBreakPointHit and OnPicaResume methods.
-     * @warning All BreakPointObservers need to be on the same thread to guarantee thread-safe state
+     * To implement event handling, override the OnPicaBreakPointHit and
+     * OnPicaResume methods.
+     * @warning All BreakPointObservers need to be on the same thread to guarantee
+     * thread-safe state
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 7]
```diff
@@ -47,18 +48,23 @@ class DebugContext {
...
      * access
-     * @todo Evaluate an alternative interface, in which there is only one managing observer and
+     * @todo Evaluate an alternative interface, in which there is only one
+     * managing observer and
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 8]
```diff
@@ -47,18 +48,23 @@ class DebugContext {
...
      * multiple child observers running (by design) on the same thread.
      */
     class BreakPointObserver {
     public:
-        /// Constructs the object such that it observes events of the given DebugContext.
+        /// Constructs the object such that it observes events of the given
+        /// DebugContext.
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 9]
```diff
@@ -83,19 +91,22 @@ class DebugContext {
          * Action to perform when a breakpoint was reached.
          * @param event Type of event which triggered the breakpoint
          * @param data Optional data pointer (if unused, this is a nullptr)
-         * @note This function will perform nothing unless it is overridden in the child class.
+         * @note This function will perform nothing unless it is overridden in the
+         * child class.
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad

File: src/video_core/debug_utils/debug_utils.h [hunk 10]
```diff
@@ -83,19 +91,22 @@ class DebugContext {
...
          */
         virtual void OnPicaBreakPointHit(Event, void*) {}
 
         /**
          * Action to perform when emulation is resumed from a breakpoint.
-         * @note This function will perform nothing unless it is overridden in the child class.
+         * @note This function will perform nothing unless it is overridden in the
+         * child class.
```
Comment (JayFoxRox) [inline src/video_core/debug_utils/debug_utils.h]: bad (rest of file too)

**Comment by JayFoxRox** [comment]
This is the latest (rebased) version of it @jphalimi
I still didn't personally test it as my notebook broke. If this is confirmed working I'll move it back to my real GS WIP branch (#2). I'll then do some more style fixes and create seperate PR for individual pieces.

There is still a known regression in some games + fincs documented proper behaviour with the b15 register. There is also still a known bug about initialization which spams logs and might break a couple of initial draw calls.

---

Ignore any PRs which are not by me, I simply didn't have my master up-to-date when pushing to this

**Comment by jphalimi** [comment]
Dear @JayFoxRox, is there a reason why you deleted this branch? Is your refactoring cancelled?

**Comment by JayFoxRox** [comment]
See #2. This was merely a backup in case something with clang-format would have gone wrong

**Pull request closed without merging**